maintenance/aws-janitor/main.go

```diff
@@ -27,15 +27,19 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/iam"
-	"github.com/golang/glog"
+	"k8s.io/klog"
 	"k8s.io/test-infra/maintenance/aws-janitor/resources"
 	s3path "k8s.io/test-infra/maintenance/aws-janitor/s3"
 )
 
-const defaultRegion = "us-east-1"
+const (
+	defaultRegion = "us-east-1"
```

Comment:
any particular reason for this?

Reply:
I think this is one of the tool vscode has on save that reorders this and imports stmts.